### ISSUE TITLE
Unable to remove tasks from unfinished list

### ISSUE THREAD

**Issue by hatharom** [Codewars/codewars.com#289]
I have java as preffered language and filter accordingly , however the filter didnt worked because I got tasks from other language. I want to remove those from the unfinished list.
thanks

**Comment by AWulkan** [comment]
I would also like to be able to do this. If I just click on a link that goes to a kata it will be added to my unfinished list, even though I never intended to do it. It's pretty annoying.

**Comment by NeuroSerpens** [comment]
So... Will such a feature be implemented? I'm guessing it would be quite simple to just add a button and delete stuff related to your user in the database...

**Comment by jhoffner** [comment]
Is this feature really necessary? There is already the bookmarking and collection features. Either of those would be (and are intended to) help you keep track of kata you wish to revisit later.

**Comment by NeuroSerpens** [comment]
Well... I would like it. And I think these two othet people also would. And
since (at least I think) it should be simple... Why not? Also as you said
there are other options to keep track of tasks too anyway.

**Comment by NeuroSerpens** [comment]
The way I see it, as the unfinished section is public it also kind of
taints the profile.

**Comment by jhoffner** [comment]
Good point on the profile. In the next release that will be a private list only. 

Its definitely not a trivial feature to add. Some sort of UX needs to be implemented and there are edge cases I can think of. Making the unfinished list only update if you actually attempt the kata makes sense (it should have been this way to begin with, might be a bug).

**Comment by NeuroSerpens** [comment]
Great. That solves it.
Immense gratitude.

**Comment by chemicalwill** [comment]
I don't suppose there is yet a way to do this? Somehow opened a Haskell-only kata and I only know a little Python...

**Comment by blzzua** [comment]
It looks like a incentive to learn that language, will be to remove such tasks from unfinished.

**Comment by vsevolod19860507** [comment]
Any news?
